yw_website/yw_website/urls.py

```diff
@@ -32,7 +51,13 @@
     path('home/users/', script_upload_views.users, name = 'users'),
     path('upload/', script_upload_views.model_form_upload, name='upload'),
     #test path- remove later
-    path('home/test', script_upload_views.home, name='test')
+    path('home/test', script_upload_views.home, name='test'),
+
+    #REST API URLS
+    path('api/', include(router.urls)),
+
+
+    # path('api-auth/', include('rest_framework.urls')), # This is for rest stuff that should be hidden behind authentication.
```

Comment:
should this commented code be left in?

Reply:
I left it in because I'm not sure how much of our API will be authenticated and not authenticated. This would help us require someone to be logged in to use the API.